stylist/context_processors.py

```diff
@@ -44,7 +44,7 @@ def get_custom_styles(request):
         custom_style = None
         custom_font_import = None
     try:
-        if getattr(settings, 'STYLIST_IGNORE_SASS', False):
+        if getattr(settings, 'STYLIST_IGNORE_SASS', True):
```

Comment:
This would be a bit cleaner with the other naming convention and with out so many "nots"